### PR TITLE
Delete Gnocchi endpoint from Wallaby deployment

### PR DESCRIPTION
**Pull request by yadneshk**

### CODE REVIEW

**Comment by cescgina** [comment]
recheck

File: docs_user/modules/proc_adopting-telemetry-services.adoc [hunk 1]
```diff
@@ -14,6 +14,13 @@ This guide also assumes that:
 * Previous Adoption steps completed. MariaDB, the {identity_service_first_ref} and the data plane should be already adopted.
 //kgilliga:Should this procedure be moved after the "Adopting the data plane" chapter?
 
+* Delete Gnocchi endpoint from previous deployment, if it exists
++
+----
+oc rsh openstackclient openstack endpoint list | grep gnocchi
+oc rsh openstackclient openstack endpoint delete gnocchi
```
Comment (jistr) [inline docs_user/modules/proc_adopting-telemetry-services.adoc:21]: Can we instead add gnocchi to this list? https://github.com/openstack-k8s-operators/data-plane-adoption/blob/6c16b7f29eafc829b2dcaec4c41dac1925f505ab/docs_user/modules/proc_adopting-the-identity-service.adoc#L66 

That should delete both the service and the endpoint (that's how we delete endpoints for the other services).

**Review by jistr — CHANGES_REQUESTED** [review]

**Comment by yadneshk** [comment]
recheck

File: docs_user/modules/proc_adopting-telemetry-services.adoc (hunk 1 above)
Comment (yadneshk) [reply docs_user/modules/proc_adopting-telemetry-services.adoc]: Thank for, I've made changes according to it.

**Review by jistr — APPROVED** [review]
:+1: Thank you

**Pull request merged**